app/controllers/recipes_controller.rb

```diff
@@ -1,6 +1,9 @@
 class RecipesController < ApplicationController
+  before_action :authenticate_user!
+
   def index
     @recipes = current_user.recipes
+    @user_id = current_user.id
   end
 
   def new
```

Comment:
`def destroy` method redirects to a url that doesn't exist.

Kindly correct it to: `user_recipes_url`